dashboard/app/models/user.rb

```diff
@@ -1070,14 +1070,12 @@ def self.progress_queue
 
   # can this user edit their own account?
   def can_edit_account?
-    return true if teacher? || encrypted_password.present? || oauth?
-
-    # sections_as_student should be a method but I already did that in another branch so I'm avoiding conflicts for now
```

Comment:
`sections_as_student` is available on the model now, as called out by this comment.